zio-http/shared/src/main/scala/zio/http/Response.scala

```diff
@@ -133,12 +133,14 @@ object Response {
 
   def badRequest(message: String): Response = error(Status.BadRequest, message)
 
-  def error(status: Status.Error, message: String): Response = {
+  def badRequest(message: String, includeWarning: Boolean): Response = error(Status.BadRequest, message, includeWarning)
+
+  def error(status: Status.Error, message: String, includeWarning: Boolean = false): Response = {
     import zio.http.internal.OutputEncoder
 
     val message2 = OutputEncoder.encodeHtml(if (message == null) status.text else message)
 
-    Response(status = status, headers = Headers(Header.Warning(199, "ZIO HTTP", message2)))
+    Response(status, body = if (includeWarning) Body.fromString(s"warning: $message2") else Body.empty)
```

Comment:
Added a `"warning: "` prefix because on [`http/HandlerAspect.scala:838`](https://github.com/zio/zio-http/pull/2897/files#diff-319b46aec125e01353188fe3d4b91b66d67c301dee58a03ed0d3510b228bb472R838) and [`http/HandlerAspect.scala:885`](https://github.com/zio/zio-http/pull/2897/files#diff-319b46aec125e01353188fe3d4b91b66d67c301dee58a03ed0d3510b228bb472R885) we used the warning header to build respectively an error body and a console error message, and IDK if it'd be possible to have a non-warning on the body at that step in the code.